README.es.md

```diff
@@ -0,0 +1,121 @@
+# sway [![](https://api.travis-ci.org/swaywm/sway.svg)](https://travis-ci.org/swaywm/sway) [![Donate with fosspay](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Español](https://github.com/swaywm/sway/blob/master/README.es.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--)
+
+"**S**irCmpwn's **Way**land compositor" es un gestor de composición de ventanas
+compatible con i3 [Wayland](http://wayland.freedesktop.org/) **en processo de
+desarrollo**.
+Lee el ["FAQ"](https://github.com/swaywm/sway/wiki). Únete al canal de
+["IRC"](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway en
+irc.freenode.net).
+
+**Nota**: Sway se encuentra actualmente "congelado"("frozen"), es decir que
```

Comment:
`"congelado"("frozen")` seems missing a space between the `"` and `(`. I'd expect to have one there.

Reply:
fixed